docs/Makefile

```diff
@@ -0,0 +1,177 @@
+# Makefile for Sphinx documentation
```

Comment:
I doubt, they would need gradle at readthedocs end :) This documentation is not build during axion-release build.